### ISSUE TITLE
requests traces show up under django service as an operation instead of as its own service

### ISSUE THREAD

**Issue by siva273** [DataDog/dd-trace-py#2211]
We have ddtrace set up using ddtrace-run and service name set up using DD_SERVICE_NAME(?).
Each service is given an unique color for better visualisation and understanding.

<img width="482" alt="Screenshot 2021-03-23 at 2 50 10 PM" src="https://user-images.githubusercontent.com/18142532/112127010-04db7a00-8beb-11eb-8549-019610df8049.png">

Now there are external requests happening using the requests library.
Currently it shows up as service django and operation requests.request
However we want it to show up as requests and operation requests.request
so that we can color requests differently and understand how much time is spent inside django and how much in external requests from django.

<img width="1069" alt="Screenshot 2021-03-23 at 3 11 26 PM" src="https://user-images.githubusercontent.com/18142532/112127147-23da0c00-8beb-11eb-8c9f-bd76f3341499.png">

@codebreach

**Comment by codebreach** [comment]
To add to this, the challenge is that the `requests` calls are showing as an operation under the service `django` however we want it to show up as its own service so that we can color it and all.

To do this we added the following code. However it doesnt seem to be doing anything

```python
from ddtrace import config

# make requests use a different service name so its easier to see in the UI
config.requests["service_name"] = "requests"
```

**Comment by Kyle-Verhoog** [comment]
Hi @siva273, @codebreach!

Thanks for the report and apologies for the delay.

Yeah it looks like the requests integration service naming logic was incorrect. I have opened up a fix: #2273.

@codebreach as for your code example, this would work typically but the requests integration did not yet have support for the global config API (I have also added this in #2273).

The work around in the mean-time would be to use the session-based configuration listed in the docs: https://ddtrace.readthedocs.io/en/stable/integrations.html#requests although this isn't very ideal as you'd have to set it for each session.
